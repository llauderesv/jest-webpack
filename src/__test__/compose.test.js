// import { compose } from 'redux';

// describe('Compose test suite', () => {
//   it('Should return value using compose function', () => {
//     function f0() {
//       return () => 1;
//     }

//     function f1(b) {
//       return () => b() + 1;
//     }

//     function f2(a) {
//       return () => a() + 1;
//     }

//     const enhance = compose(
//       f1,
//       f2,
//     );

//     const enhanceResult = enhance(f0());

//     expect(enhanceResult()).toMatchSnapshot();
//   });
// });
