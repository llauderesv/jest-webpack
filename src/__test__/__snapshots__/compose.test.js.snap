// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compose test suite Should return value using compose function 1`] = `6`;
